### PR TITLE
Chocolate Doom: Added recipe for 2.2.1

### PR DESCRIPTION
**Pull request by lich**

### CODE REVIEW

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 1]
```diff
@@ -0,0 +1,53 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. The purpose of \
```
Comment (humdingerb) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: make sure lines arean't longer than 80 chars

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+SOURCE_URI="http://www.chocolate-doom.org/downloads/$portVersion/chocolate-doom-$portVersion.tar.gz"
+SOURCE_DIR="chocolate-doom-$portVersion"
+CHECKSUM_SHA256="ad11e2871667c6fa0658abf2dcba0cd9b26fbd651ee8df55adfdc18ad8fd674a"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chocolate_doom = $portVersion
+	app:chocolate_doom = $portVersion
```
Comment (humdingerb) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: if the resulting binary is installed in $binDir, it should be cmd:chocolate_doom

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe (hunk 1 above)
Comment (DarkmatterVale) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: You can use "\" to continue displaying a single line on multiple lines.

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	cmd:make
+	cmd:pkg_config
+	cmd:python
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+        make $jobArgs
```
Comment (humdingerb) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: only one tab indention

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+INSTALL()
+{
+        cd src
```
Comment (humdingerb) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: only one tab indention

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+	mkdir -p $binDir
+        cp chocolate-doom chocolate-doom-setup chocolate-hexen chocolate-hexen-setup chocolate-heretic chocolate-heretic-setup chocolate-server chocolate-setup chocolate-strife-setup chocolate-strife $binDir/
```
Comment (humdingerb) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: only one tab indention. also use line breaks with "\" and indent the new lines with another tab.

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 6]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="http://www.chocolate-doom.org"
+COPYRIGHT="Chocolate Doom Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.chocolate-doom.org/downloads/$portVersion/chocolate-doom-$portVersion.tar.gz"
+SOURCE_DIR="chocolate-doom-$portVersion"
```
Comment (DarkmatterVale) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: Do you ever use this? If not, you should remove the line.

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe [hunk 7]
```diff
@@ -0,0 +1,63 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. \
+The purpose of Chocolate Doom is to be as compatible as possible with the \
+original DOS version of Doom."
+HOMEPAGE="http://www.chocolate-doom.org"
+COPYRIGHT="Chocolate Doom Team"
```
Comment (scottmc) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: Is there any copyright years listed?

**Comment by scottmc** [comment]
this should go in games-fps as it is in gentoo-portage

File: games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe (hunk 7 above)
Comment (pulkomandy) [inline games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe]: Also, iD software still has copyright on some of the code. Please research the copyright owners.

**Pull request merged**